modules/actions/pool/staking/sync-gauge-staking.service.ts

```diff
@@ -193,7 +193,7 @@ export const syncGaugeStakingForPools = async (
     const allStakingGaugeRewards = allDbStakingGauges.map((gauge) => gauge?.rewards).flat();
 
     // DB operations for gauge reward tokens
-    for (const { id, rewardPerSecond } of onchainRates) {
+    for (const { id, rewardPerSecond, isDirectRewardToken } of onchainRates) {
```

Comment:
what does it mean direct reward? anything by emissions? do we need that, since there is emissions type?